Controllers/ViewModels/TodoitemViewModel.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Globalization;
+using Taskmanager.Data.Entities;
+
+namespace Taskmanager.Controllers.ViewModels
+{
+    public class TodoitemViewModel
+    {
+        public string Title { get; set; }
+        public bool IsFinished { get; set; }
+        public DateTime DeadlineDate { get; set; }
+        public long? PriorityId { get; set; }
+
+        public static TodoitemViewModel MapTodoitem(Todoitem todoitem)
+        {
+            if (todoitem != null)
+            {
+                return new TodoitemViewModel
+                { 
+                    Title = todoitem.Title, 
+                    IsFinished = todoitem.Isfinished == 1 ? true : false, 
```

Comment:
Not necessary expression, you can either use (bool)todoItem.IsFinished or todoItem.IsFinished == 1 (better usage and it's bool by result, no need to compare with true or false)